04/app.js

```diff
@@ -12,8 +12,13 @@ class App extends React.Component {
     }
 
     renderUsersList() {
-        const {users} = this.state;
-        return users.map(name => {
+        const {users, searchQuery} = this.state;
+        let userList = users;
+        if(searchQuery) {
+            userList = users.filter(name=> name.includes(searchQuery));
```

Comment:
Warto pominąć wielkość liter - można to zrobić przez np. sprowadzenie `name` oraz `searchQuery` do zapisu tylko przy wykorzystaniu mały liter tj. `.toLowerCase`